src/validatePrTitle.test.js

```diff
@@ -55,11 +55,25 @@ describe('defined scopes', () => {
     await validatePrTitle('fix(core): Bar', {scopes: ['core']});
   });
 
+  it('allows multiple matching scopes', async () => {
+    await validatePrTitle('fix(core,e2e): Bar', {
+      scopes: ['core', 'e2e', 'web']
+    });
+  });
+
+  it('throws when an unknown scope is detected within multiple scopes', async () => {
+    await expect(
+      validatePrTitle('fix(core,e2e,foo,bar): Bar', {scopes: ['foo', 'core']})
```

Comment:
any issue in convention of comma separator symbol `,` with—or without—a trailing space?

Reply:
Not sure about the official convention, but I think we should support spaces in this action:

https://github.com/amannn/action-semantic-pull-request/blob/1829a7c0a510f3f783b5463c8c05923802e7de3a/src/validatePrTitle.js#L62